packages/api-client/src/notification/NotificationAPI/NotificationAPI.ts

```diff
@@ -108,6 +108,10 @@ export class NotificationAPI {
             hasMissedNotifications = true;
             payload = {...defaultPayload, ...error.response?.data};
           }
+        } else if (error instanceof BackendError && error.label === BackendErrorLabel.NOT_FOUND) {
+          //notification was not found in the database,
+          //we need to load all the notifications from the beginning (without 'since' param)
+          return getNotificationChunks(currentClientId);
```

Comment:
Since we're fetching all the notifications, should we also reset `notificationList` array to empty array?

Reply:
hummm does this returns a payload that says `{missedNotifications: true, ...}`?

Can you maybe write a test case for both scenarios?

Reply:
Sure!